README.md

```diff
@@ -388,6 +409,55 @@ Only expose foreign keys for the comment model findById method. eg. `GET /api/co
 - Type: `boolean|array`
 - Default: `false`
 
+## Custom remote
+
+### `jsonapi` remote options
+Sometime you need to control if custom remote should be serialized or not.
+**By default a custom remote will NOT be handled by JSONApi.**
+
+To enabled/disable JSONApi for specific remotes, you have multiple solutions:
+
+1. Use `jsonapi` remote option
+2. Use `exlude` on component options (see above)
+3. Use `include` on component options (see above)
+4. Use `handleCustomRemote` on component options (see above)
+
+**This option has precedence** and it forces the remote to BE or to NOT BE deserialized/serialized by JSONApi.
```

Comment:
This option takes precedence and sets the component to handle or not handle the custom remote method.